src/test/java/com/serenitydojo/WhenCreatingObjects.java

```diff
@@ -4,4 +4,20 @@
 import org.junit.Test;
 
 public class WhenCreatingObjects {
+    @Test
+    public void creating_a_dog(){
+        Dog fido = new Dog("Fido", "Bone", 5);
+        Assert.assertEquals(fido.getName(), "Fido");
+        Assert.assertEquals(fido.getFavoriteToy(), "Bone");
+
+//        Commenting sample code because compiler cannot decide which one of the assertEquals flavor to call:
+//        java: reference to assertEquals is ambiguous
+//        both method assertEquals(java.lang.Object,java.lang.Object) in org.junit.Assert and method assertEquals(long,long) in org.junit.Assert match
+//        Assert.assertEquals(fido.getAge(),5);
+
+//      Using some google search help, manipulated the second arg ("expected value")
+//      The result is an Integer object that represents the integer value specified by the string.
+        Assert.assertEquals(fido.getAge(), Integer.valueOf("5"));
```

Comment:
Hi @wakaleo 
I feel like sometimes I need to manipulate/ override the already provided code like I did in this case. I don't mind it except I don't know if there was a way to do this without needing to. Can you please review?

Reply:
Hi Ash, thanks for letting me know: that could be an error in the GitHub classroom setup - if you let me know when you see that happen, we can rectify it.

Reply:
You can simplify this by writing:
```
 Assert.assertEquals(fido.getAge(), 5);

```

Reply:
Thanks for looking @wakaleo ! The line `Assert.assertEquals(fido.getAge(), 5);` throws error the moment I finish typing it - so before compilation itself and of course an error comes up after I run it. The junit assert package import appears to be right. I have attached a gif for you to see.
![Assert on integer](https://github.com/serenity-dojo/exercise-3-ashwiniraokarai/assets/29970871/f257e5a0-df94-4318-92ba-7bd55203bcd3)

Here's something to know:
Inside the Dog class, I've defined the _age_ attribute is an Integer Object) type, not primitive. Accordingly _getAge()_ returns an Integer Object. Changing the type to _primitive_ no longer throws error. 

While this great for getting past the issue, I'd like to understand how to make this work while retaining the Integer type for _age_ (the actual value on the Assert)

Also, digging further into the Assert statement where I pass along the expected value, a literal value of `5`. How do I know what type I am passing (primitive? Integer?). I am interesting in learning how to be able to find that out. If it were a variable, I could do .getClass().getName() on it but in this case it's a literal value.